### PR TITLE
NODE-89: MarkLogic module v 2.9.1 for Node.js at Windows OS does not work

### PR DESCRIPTION
**Pull request by mariusmarklogic**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     "test:setup": "node etc/test-setup.js",
     "test:teardown": "node etc/test-teardown.js",
     "test": "gulp test",
-    "preinstall": "([ ! -f package-lock.json ] && npm install --package-lock-only --ignore-scripts); yes | npx npm-force-resolutions"
+    "preinstall": "npx npm-force-resolutions"
```
Comment (anu3990) [inline package.json:14]: Hi @mariusmarklogic , i tried this in my local workspace and it threw the following - "Error: ENOENT: no such file or directory, open './package-lock.json'"
Please try the following -
1. git clone https://github.com/marklogic/node-client-api.git
2. git checkout develop
3. vi package.json -  change the preinstall line to -  "preinstall": "([ ! -f package-lock.json ] && npm install --package-lock-only --ignore-scripts); npx npm-force-resolutions" 
4. Run - npm install
5. mocha test-basic --no-timeouts

**Review by anu3990 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (anu3990) [reply package.json:14]: Please let me know if the above steps worked in your workspace. Thanks.!

Comment (mariusmarklogic) [reply package.json:14]: I tried "npm install" and it didn't worked. It worked with the workaround "--ignore-scripts".

Comment (anu3990) [reply package.json:14]: Thanks for the update @mariusmarklogic .! What was the error you got on running - npm install?

**Pull request closed without merging**